react-api/src/components/BookList/BookList.tsx

```diff
@@ -0,0 +1,66 @@
+import classnames from '../../utils/classnames'
+import s from './BookList.module.scss'
+import { BookTitle } from '../../API/openLibrary.org/search'
+import Loader from '../Loader/Loader'
+import NoResultSvg from './no-result.svg'
+import Pagination from '../Pagination/Pagination'
+
+const cx = classnames.bind(s)
+
+type IBookList = {
```

Comment:
Please, consider a reading... what's the difference between interface and type in TypeScript.